okhttp/src/main/java/com/backblaze/b2/client/okHttpClient/B2StorageOkHttpClientBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2017, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+package com.backblaze.b2.client.okHttpClient;
+
+import com.backblaze.b2.client.B2AccountAuthorizer;
+import com.backblaze.b2.client.B2AccountAuthorizerSimpleImpl;
+import com.backblaze.b2.client.B2ClientConfig;
+import com.backblaze.b2.client.B2DefaultRetryPolicy;
+import com.backblaze.b2.client.B2RetryPolicy;
+import com.backblaze.b2.client.B2Sdk;
+import com.backblaze.b2.client.B2StorageClient;
+import com.backblaze.b2.client.B2StorageClientImpl;
+import com.backblaze.b2.client.B2StorageClientWebifier;
+import com.backblaze.b2.client.B2StorageClientWebifierImpl;
+import com.backblaze.b2.client.credentialsSources.B2Credentials;
+import com.backblaze.b2.client.credentialsSources.B2CredentialsFromEnvironmentSource;
+import com.backblaze.b2.client.webApiClients.B2WebApiClient;
+
+import java.util.function.Supplier;
+
+public class B2StorageOkHttpClientBuilder {
+
+    private static final String DEFAULT_MASTER_URL = "https://api.backblazeb2.com/";
+
+    private final B2ClientConfig config;
+    private Supplier<B2RetryPolicy> retryPolicySupplier;
+
+    @SuppressWarnings("WeakerAccess")
+    public static B2StorageOkHttpClientBuilder builder(String userAgent)  {
```

Comment:
now that i have added B2StorageClientFactory and it has a method that does this, perhaps you don't need it here?  I don't think I can remove it from the Apache HttpClient implementation because we've been shipping that way for years, but maybe we don't need to add it here?

What do you think?  Let's see what you come up with after looking at the new B2StorageClientFactory code.

Reply:
what do you think, @OldSkoolMark.  should we ditch this and the one i marked below?

Reply:
If http-client has it I think we okhttp should too. That makes it easier for any client code that uses  those static methods to switch to the other http client.